.vscode/schema/manifest.webextension-shim.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+        "external-uri": {
+            "type": "array",
+            "description": "external uri (regex based)",
+            "items": {
+                "type": "string"
+            }
+        },
+        "user-scripts": {
+            "type": "array",
+            "description": "user scripts file path list",
+            "items": {
+                "type": "string"
+            }
+        }
+    },
+    "required": ["external-uri", "user-scripts"]
```

Comment:
They're all optional in API perspective

Reply:
Why?

Reply:
What if they don't have anything external? What if they don't have user-scripts?